internal/jimm/jimm.go

```diff
@@ -195,9 +196,8 @@ type SSHManager interface {
 	// PublicKeyHandler is the method to verify the public key of the user. It returns a user if successful.
 	PublicKeyHandler(ctx context.Context, claimUser string, key []byte) (*openfga.User, error)
 
-	// ControllerInfoFromModelUUID is the method to resolve the address of the controller to contact given the model UUID and
-	// a valid JWT To connect to the controller.
-	ControllerInfoFromModelUUID(ctx context.Context, modelUUID string, user *openfga.User) (ssh.ControllerInfo, error)
+	// DialControllerSSHServer dials the controller hosting the specific model UUID.
+	DialControllerSSHServer(ctx context.Context, modelUUID string, user *openfga.User) (*gossh.Client, error)
```

Comment:
I also really hate how we pass users as extra arguments, we do it so often that we may as well a) put it on context with a specific key and ALWAYS expect a user to be there or have a new type which contains the user. Not for this PR but we should discuss this. Passing users as arguments is really ugly and annoying. It also couples our signatures and users to callers.